tests/test_acasclient.py

```diff
@@ -2296,6 +2296,144 @@ def test046_cmpdreg_admin_crud(self):
         self.client.delete_cmpdreg_vendor(vendor['id'])
         
 
+    def test_047_load_sdf_with_salts(self):
+        """
+        Test to Load SDF File w/ Salts to Determine Process
```

Comment:
nit: I think you can update this docstring now that you understand what this is testing